### PR TITLE
included molecule role paths in molecule_workflow.yml action

### PR DESCRIPTION
**Pull request by Infinity-ops**
So far,  the molecule test is implemented only for three roles. Hence specifying those paths in molecule_workflow.yml reduces github action load in the future.

### CODE REVIEW

**Comment by enasca** [comment]
This is already an improvement, but do you think we can go further and introduce some parallelism? At the moment, we're iterating over the molecule directories with `find` in a strictly sequential fashion. If we were able to execute all tests in parallel, we would only be limited by the performance of the slowest test.

From a quick glance at the github docs, https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs looks like a good starting point.

**Comment by Infinity-ops** [comment]
Found this docs useful, Thanks for sharing.
I have implemented the workflow using a single-dimension matrix strategy. Your comments would be helpful if more improvement was required.

File: ansible/roles/proxies/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 proxy_env_filename: proxy.sh
 http_proxy_host: proxy.example.docker.de
-http_proxy_port: "3128"
+http_proxy_port: "3129"
```
Comment (enasca) [inline ansible/roles/proxies/defaults/main.yml:3]: Is this change intentional?

**Review by enasca — CHANGES_REQUESTED** [review]

File: ansible/roles/proxies/defaults/main.yml (hunk 1 above)
Comment (Infinity-ops) [reply ansible/roles/proxies/defaults/main.yml:3]: Yes, it's modified to make it possible to trigger the molecular action workflow.
Thanks for the reminder, I will change it back.

**Review by enasca — APPROVED** [review]
